README.md

```diff
@@ -160,3 +166,26 @@ apiClient.AuthenticateUserAsync("username", "password", new Response<Authenticat
 	}
 });
 ```
+
+---
+
+## Reference Use-Cases
```

Comment:
```suggestion
## Projects using the API client
```

Reply:
I tried thinking of a proper headline and could not think of that xD